### PR TITLE
Separate webhooks from cluster operator

### PR DESCRIPTION
**Pull request by fredtibbitts**
Separate the validating webhooks from the cluster operator into a separate pod. This facilitates HA for the cluster operator.

### CODE REVIEW

File: cluster-operator/internal/certificate/certificate.go [hunk 1]
```diff
@@ -16,74 +16,75 @@ import (
...
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 )
 
 const (
 	// OperatorName is the resource name for the Verrazzano cluster operator
-	OperatorName = "verrazzano-cluster-operator"
+	OperatorName = "verrazzano-cluster-operator-webhook"
```
Comment (desagar) [inline cluster-operator/internal/certificate/certificate.go:29]: minor nit - this is not really an "Operator" name but a webhook name

File: cluster-operator/internal/operatorinit/update_webhooks.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	"context"
+
+	"github.com/verrazzano/verrazzano/cluster-operator/internal/certificate"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// deleteValidatingWebhookConfiguration deletes a validating webhook configuration
+func deleteValidatingWebhookConfiguration(kubeClient kubernetes.Interface, name string) error {
```
Comment (desagar) [inline cluster-operator/internal/operatorinit/update_webhooks.go:16]: not sure where this is used

Comment (fredtibbitts) [reply cluster-operator/internal/operatorinit/update_webhooks.go]: I deleted this function and the associated unit tests. The VAO does not have a delete function but the VPO does have a delete function. Lots of inconsistencies in the way our operators deal with webhooks.

File: cluster-operator/internal/certificate/certificate.go (hunk 1 above)
Comment (fredtibbitts) [reply cluster-operator/internal/certificate/certificate.go]: I renamed this and the namespace constant as well.

**Review by davisgomes — APPROVED** [review]

**Pull request merged**